### PR TITLE
Fix display of bbch_growth_range, and hide dummy details from dataset_list

### LINKED ISSUE #320: Clicking on any dataset link opens blank white page
Dataset uploaded fine, went to review stage but clicking on the dataset name briefly (<0.5s) flicked up some page before going to a blank white page. Can accept/reject but can't see any data.

### LINKED ISSUE #321: Task Type and Crop default to classification and pasture?
Looks like the task type and the crop categories in the review phase default to classification and pasture. Is this a hangover of the DeepWeeds dataset?

Pasture isn't mentioned in the weedcoco.json associated with the upload.

### PR DESCRIPTION
**Pull request by jnothman**
Fixes #320
Fixes #321

### CODE REVIEW

File: search/src/Components/dataset/dataset_summary.js [hunk 1]
```diff
@@ -51,6 +51,10 @@ const baseURL = new URL(window.location.origin);
 
 const AgContextFieldList = (props) => {
     const {title, agcontext, fields, classes, ...accordionProps} = props;
+    const formatters = {
+      bbch_growth_range: (val) => (val["min"] ? val["min"] + " to " + val["max"] : val),
```
Comment (ElevnLi) [inline search/src/Components/dataset/dataset_summary.js:55]:
```suggestion
      bbch_growth_range: (val) => (val["min"] && val["max"] ? val["min"] + " to " + val["max"] : val),
```

Comment (jnothman) [reply search/src/Components/dataset/dataset_summary.js:55]: If we trust that it's valid according to the schema, then the value is either "na" or {"min": int, "max": int}:

https://github.com/Sydney-Informatics-Hub/Weed-ID-Interchange/blob/f11e6bb3ca96ed6714d1b13144c995c87c5698ba/weedcoco/schema/AgContext.yaml#L59-L79

**Comment by jnothman** [comment]
I'm currently building with this fix in the deploy branch, btw...

**Comment by ElevnLi** [comment]
> I'm currently building with this fix in the deploy branch, btw...

No.. Deploy branch is deprecated. Just build on this branch or merge it to master and build on master

**Comment by jnothman** [comment]
Okay, not in deploy... just on the server :)

**Pull request merged**